main/lsp/LSPConfiguration.h

```diff
@@ -37,6 +37,9 @@ class LSPClientConfiguration final {
     /** If true, then LSP outputs a warning for untyped values */
     bool enableHighlightUntyped = false;
 
+    /** If false, nudges in `typed: false` files are disabled */
+    bool enableNudges = true;
```

Comment:
Can we call this "enableTypedFalseNudges"? or maybe even "enableTypedFalseCompletionNudges" because at the moment, you're not using this flag to e.g. disable the hover suggestions

Reply:
Sure will do. Thanks for the speedy review.